meilisearch-http/src/routes/stop_words.rs

```diff
@@ -1,63 +1,83 @@
-use std::collections::BTreeSet;
-
+use actix_web::{web, HttpResponse};
+use actix_web_macros::{delete, get, post};
 use meilisearch_core::settings::{SettingsUpdate, UpdateState};
-use tide::{Request, Response};
+use std::collections::BTreeSet;
 
-use crate::error::{ResponseError, SResult};
-use crate::helpers::tide::RequestExt;
-use crate::helpers::tide::ACL::*;
-use crate::routes::document::IndexUpdateResponse;
+use crate::error::ResponseError;
+use crate::helpers::Authentication;
+use crate::routes::{IndexParam, IndexUpdateResponse};
 use crate::Data;
 
-pub async fn get(ctx: Request<Data>) -> SResult<Response> {
-    ctx.is_allowed(Private)?;
-    let index = ctx.index()?;
-    let db = &ctx.state().db;
-    let reader = db.main_read_txn()?;
+pub fn services(cfg: &mut web::ServiceConfig) {
+    cfg.service(get).service(update).service(delete);
```

Comment:
```suggestion
    cfg.service(get)
        .service(update)
        .service(delete);
```

This feels easier to read.

Reply:
For `meilisearch-http`, I have respected the `cargo fmt` format. Do you think we must stop respect it or add an exception for this one?

Reply:
Keep it like you have done, in one line, that's ok.